backend/src/test/java/com/javahelp/backend/data/DynamoDBUserStoreTest.java

```diff
@@ -140,6 +140,46 @@ public void testPasswordUpdateRead() {
         db.delete(u.getStringID());
     }
 
+    @Test
```

Comment:
Is it OK not to have timeout time?

Reply:
Good idea, that definitely makes sense here!